addons/website/views/snippets/s_faq_horizontal.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_faq_horizontal" name="Topics List">
+    <section class="s_faq_horizontal">
+        <div class="container">
+            <div class="row">
+                <div data-name="Heading" class="col-lg-6 offset-lg-3 pt48 pb48">
+                    <h2 style="text-align:center;">Topic Walkthrough</h2>
+                    <p style="text-align:center;">Learn how to quickly set up and start using our services with our step-by-step onboarding process.</p>
+                </div>
+            </div>
+            <div class="row s_col_no_resize">
+                <div data-name="Topic" class="s_faq_horizontal_entry col-12 mb-2 pt16 pb32">
+                    <article class="row">
+                        <hgroup class="col-lg-4" role="heading">
+                            <div class="s_faq_horizontal_entry_title position-lg-sticky pb-lg-3 transition-base overflow-auto" style="top: 16px">
+                                <h3 class="h5-fs">Getting Started</h3>
```

Comment:
When I assign the style using the editor's toolbar, I get `h5-fs` within a nested `<span>`.

Reply:
Yes but... using a nested `<span>` will invalidate user edition. Are we sure that's the expected behaviour ?

https://github.com/user-attachments/assets/b7477386-972f-4864-bc4e-64b56d29adf4

Reply:
@bso-odoo No nested span, let's put the class on the hx tags: https://discord.com/channels/678381219515465750/818823260967010304/1278692494451675238

Reply:
Both lead to the same situation after being edited, so, it's fine by me the way you did.
The editor side will definitely have to be fixed.